tests/functional/services/auth/service_client/test_get_identities.py

```diff
@@ -55,15 +55,15 @@ def test_get_identities_success(usernames, service_client):
     [
         (True, "true"),
         (False, "false"),
-        (1, "true"),
-        (0, "true"),
-        ("fALSe", "false"),
-        ("true", "true"),
+        (None, "false"),
     ],
 )
 def test_get_identities_provision(inval, outval, service_client):
     load_response(service_client.get_identities)
-    service_client.get_identities(usernames="globus@globus.org", provision=inval)
+    if inval is not None:
+        service_client.get_identities(usernames="globus@globus.org", provision=inval)
+    else:
+        service_client.get_identities(usernames="globus@globus.org")
```

Comment:
The parametrized list makes it seem like `None` is being tested as an invalid value, but this condition is preventing that testing.

I recommend moving this `None` test to simply check default values instead. Otherwise, the condition needs to be removed so `None` can be passed as the `provision` argument.

Reply:
Yeah, I'll move it to a separate test. I also considered
```
_OMIT = object()
```
as a local sentinel for this test, but that felt similarly wonky.